src/java.base/share/classes/java/time/LocalDate.java

```diff
@@ -604,14 +604,13 @@ public boolean isSupported(TemporalUnit unit) {
     public ValueRange range(TemporalField field) {
         if (field instanceof ChronoField chronoField) {
             if (chronoField.isDateBased()) {
-                switch (chronoField) {
-                    case DAY_OF_MONTH: return ValueRange.of(1, lengthOfMonth());
-                    case DAY_OF_YEAR: return ValueRange.of(1, lengthOfYear());
-                    case ALIGNED_WEEK_OF_MONTH: return ValueRange.of(1, getMonth() == Month.FEBRUARY && isLeapYear() == false ? 4 : 5);
-                    case YEAR_OF_ERA:
-                        return (getYear() <= 0 ? ValueRange.of(1, Year.MAX_VALUE + 1) : ValueRange.of(1, Year.MAX_VALUE));
-                }
-                return field.range();
+                return switch (chronoField) {
+                    case DAY_OF_MONTH          -> ValueRange.of(1, lengthOfMonth());
```

Comment:
For the record, I hate code that is aligned like this. It makes refactoring much more noisy, as the author has to readjust all the associated lines. I also don't believe it is within the spirit of Java's traditional coding guidelines.

Reply:
Hi Stephen. Thanks for your comments. I've tried removing the excessive spacing as you suggested. Please let me know what you think